includes/definitions/discovery/apc.yaml

```diff
@@ -241,7 +241,7 @@ modules:
                     oid: sPDUOutletControlTable
                     value: sPDUOutletCtl
                     num_oid: '.1.3.6.1.4.1.318.1.1.4.4.2.1.3.{{ $index }}'
-                    descr: 'Outlet {{ $subindex0 }} State'
+                    descr: '{{ $subindex0 }} {{ sPDUOutletCtlName }}'
```

Comment:
Oddly, this seems to cause the outlet number to be duplicated most of the time.  Not sure I like it better.  Maybe a colon would help.